pkg/tfgen/docs.go

```diff
@@ -1090,7 +1090,7 @@ func (g *Generator) convertExamples(docs, name string, stripSubsectionsWithError
 		strings.Contains(docs, "```csharp") {
 		// we have explicitly rewritten these examples and need to just return them directly rather than trying
 		// to reconvert them. But we need to surround them in the examples shortcode for rendering on the registry
-		return fmt.Sprintf("{{%% examples %%}}\n%s\n{{%% //examples %%}}", docs)
+		return fmt.Sprintf("{{%% examples %%}}\n%s\n{{%% /examples %%}}", docs)
```

Comment:
Is this simpler?

```suggestion
		return docs
```

Reply:
I don't think so - I think it's possible that this is supplying short codes to other overwritten examples maybe?